app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -0,0 +1,42 @@
+package com.example.lhi06.residue;
+
+/**
+ * Created by lhi06 on 20/07/2017.
+ */
+
+public class AlbumReview {
```

Comment:
sorry i thought these would be package private, im abit confused in what cases to use which